components/compass-runtime-agent/internal/compassconnection/package_test.go

```diff
@@ -106,8 +110,18 @@ var (
 	}
 
 	kymaModelApps = []kymaModel.Application{{Name: "App-1", ID: "abcd-efgh"}}
-
-	operationResults = []kyma.Result{{ApplicationName: "App-1", ApplicationID: "abcd-efgh", Operation: kyma.Create}}
+	testCtx       = correlation.SaveCorrelationIDHeaderToContext(context.Background(), str.Ptr(correlation.RequestIDHeaderKey), str.Ptr("test-correlation-id-123"))
+
+	operationResults    = []kyma.Result{{ApplicationName: "App-1", ApplicationID: "abcd-efgh", Operation: kyma.Create}}
+	requestIDCtxMatcher = func(ctx context.Context) bool {
```

Comment:
With new matching function I see there are two different conventions used for matching mocked parameters in thests. I think both matching unctions should be uniformed and used in the same way.  So we can declare this function here with `mock.MatchedBy`

Reply:
Done ✅